### PR TITLE
Bug fix and add a new function

### PR DESCRIPTION
**Pull request by ghurone**
Missing an improment to README.md and docstring the functions.

Add new function for check if a integer number is a prime number.

### CODE REVIEW

**Comment by raquel-ruiz** [comment]
I'm going to review this pull request

File: participants/Corrêa_Ribeiro/pr_tutorial/simple_functions.py [hunk 1]
```diff
@@ -1,12 +1,27 @@
-def fibonacci(max):
```
Comment (raquel-ruiz) [inline participants/Corrêa_Ribeiro/pr_tutorial/simple_functions.py:1]: good idea!

**Review by raquel-ruiz — APPROVED** [review]
everything approved :)

**Comment by ceb8** [comment]
Thanks! 😄

**Pull request merged**